FWCore/Utilities/scripts/edmDumpClassVersion

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+import argparse
+
+import FWCore.Utilities.ClassesDefXmlUtils as ClassesDefUtils
+
+def main(args):
+    #Need to not have ROOT load .rootlogon.(C|py) since it can cause interference.
```

Comment:
As this part is also common, how about moving it to another .py file so it can be shared?

Reply:
Agreed, I'll do that.